danpitrowiski/hw1-99-bottles-of-beer/css/style.css

```diff
@@ -0,0 +1,236 @@
+/******************************************
+/* SETUP
+/*******************************************/
+
+/* Box Model Hack */
+* {
+     -moz-box-sizing: border-box; /* Firexfox */
+     -webkit-box-sizing: border-box; /* Safari/Chrome/iOS/Android */
+     box-sizing: border-box; /* IE */
+}
+
+/* Clear fix hack */
+.clearfix:after {
+     content: ".";
+     display: block;
+     clear: both;
+     visibility: hidden;
+     line-height: 0;
+     height: 0;
+}
+
+.clear {
+	clear: both;
+}
+
+.alignright {
+	float: right;
+	padding: 0 0 10px 10px; /* note the padding around a right floated image */
+}
+
+.alignleft {
+	float: left;
+	padding: 0 10px 10px 0; /* note the padding around a left floated image */
+}
+
+/******************************************
+/* BASE STYLES
+/*******************************************/
+
+body {
+     color: #000;
+     font-size: 12px;
+     line-height: 1.4;
+     font-family: 'Lato', sans-serif;
+     padding: 20px 0 0 20px;
+}
+
+
+/******************************************
+/* LAYOUT
+/*******************************************/
+
+/* Center the container */
+#container {
+	width: 960px;
+	margin: auto;
+}
+
+#content span {
+  width: 100px;
+  height: 100px;
+  display: inline-block;
+  margin: 20px;
+  background-repeat: no-repeat;
+  background-position: top left;
+}
+
+/******************************************
+/* ADDITIONAL STYLES
+/*******************************************/
+
+.center {
+  text-align: center;
+  margin: 0;
+  position: absolute;
+  top: 42%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+}
+
+.list {
+  border: 1px solid #DDD6D8;
+  background-color: #fff;
+  padding: 20px 20px 20px 20px;
+  max-width:700px;
+  margin: 20px 0;
+  border-radius: 2px;
+}
+
+.beer {
+  position: fixed;
+  bottom: 0;
+  left: 0;
+  width: 100% !important;
+  height: 100%;
+  background-color: #F6D02E;
+  border: 4px solid #BCA13E;
+  background-size: cover;
+  z-index:-999;
+}
+
+h3 {
+  color: #B7B1B3;
+  font-weight: 300;
+}
+
+h2 {
+  color: black;
+  font-size: 32px;
+  line-height: 48px;
+}
+
+.green {
+  color: #7ED321;
+  font-size: 14px;
+  font-weight: 300;
+}
+
+button {
+  padding: 20px 20px;
+  background-color: #fff;
+  border: 1px solid #DDD6D8;
+  border-radius: 4px;
+  font-size: 28px;
+}
+
+/* removing the outline around selected buttons*/
+button:focus { outline:0 !important; }
+
+button:active {
+  background-color: #4A90E2;
+}
+
+button:hover {
+  background-color: #4A90E2;
+  color: white;
+}
+
+/*******************************************/
+/* Bubbles
+/*******************************************/
```

Comment:
:champagne:

Reply:
I should have included my inspirational codepen in my code too